backend/src/main/java/capstoneexpensetracker/backend/Travelercontroller.java

```diff
@@ -0,0 +1,21 @@
+package capstoneexpensetracker.backend;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/traveler")
+@RequiredArgsConstructor
+public class Travelercontroller {
```

Comment:
Bitte auf Kamelschreibweise achten

```suggestion
public class TravelerController {
```